ClientApp/viking-vault/src/Login.tsx

```diff
@@ -120,16 +119,15 @@ class LoginForm extends React.Component<any, ILoginFormState> {
         }, 2500);
         return null;
       }
-
-      return response.json();
-
+      if(response != null) {
```

Comment:
👮‍♂️🚔JavaScript police wants you to use `!==` and `===` instead of `!=` and `==` as the first ones also checks the type of the value and is a bit safer to use.

Reply:
That was so cute!
Da' chiar mi-a scapat. Changed it.